gno.land/cmd/gnoland/README.md

```diff
@@ -6,8 +6,88 @@
     $> cd ./gno/gno.land
     $> make install.gnoland
 
-## Run `gnoland` full node
+## Option 1: Run `gnoland` full node for local development
 
     $> gnoland start
 
 Afterward, you can interact with [`gnokey`](../gnokey) or launch a [`gnoweb`](../gnoweb) interface.
+
+
+## Option 2: Run a node and sync with a Proof of Authority (POA) network
+
+ - gnoland init
+
+ - Download genesis.json from a trusted source and save it to the root-dir/config directory.
+
+ - Get the peer's node id from the node you trust.
+
+ - Start the node
+
+       $> gnoland start --persistent "node_id@peer_ip_address:port" or add the persistent_peers value in the ./testdir/config/config.toml
+
+## Option 3: Run a node as a Proof of Authority validator starting from genesis state
+
+- Initialize the config and key files.
+
+      $> gnoland init
+
+- Return the node info; we will need it to add to validator info in the genesis.json
+
+      $> gnoland node
+
+      Address: "g14t47gv3v2z3pc23g3zr39mnc99w2cplp0jhqvv"
+      Pubkey: "E5IFULgXFdS49ILgvPmO3/8chuSWfbqw3zYXaNEP+60="
+
+- Download genesis.json from a trusted source and save it to the root-dir/config directory.
+
+- Add your validator to the genesis file.
+
+      $> genesis validator add \
+       --address g14t47gv3v2z3pc23g3zr39mnc99w2cplp0jhqvv \
+       --pub-key E5IFULgXFdS49ILgvPmO3/8chuSWfbqw3zYXaNEP+60= \
+       --power 10 \
+       --name testvalidator2
+
+- Share the genesis with all trusted validators.
+
+- Get the peer's node id from the archive node you trust.
+
+- Start the node
+
+      $> gnoland start --persistent "node_id@peer_ip_address:port"
+
+  or add the persistent_peers value in the ./testdir/config/config.toml
+
+
+## Option 4: Run as an archive node starting from genesis state
+
+It's recommended to have at least two POA validator nodes running as archive nodes to bootstrap the network.
+
+Complete the steps in Option 4 and replace the last two steps with
+
+- Retrive node id and give it trusted peers.
+
+      $> gnoland node
+
+- Start the node
+
+      $> gnoland start --prune "nothing"
+
+
+## Reset `gnoland` node back to genesis state. It's suitable for running test node
+
+    $> gnoland unsafe-reset-all
```

Comment:
Please add the flag ` --keep-addr-book`

Question: In cosmos-sdk, this command is under `gaiad tendermint unsafe-reset-all`
Should we do the same ?

```
Usage:
  gaiad tendermint [command]

Available Commands:
  reset-state      Remove all the data and WAL
  show-address     Shows this node's tendermint validator consensus address
  show-node-id     Show this node's ID
  show-validator   Show this node's tendermint validator info
  unsafe-reset-all (unsafe) Remove all the data and WAL, reset this node's validator to genesis state
  version          Print tendermint libraries' version
```